app/src/main/java/org/sopt/and/data/dataremote/model/request/RequestLoginDto.kt

```diff
@@ -1,4 +1,4 @@
-package org.sopt.and.api.dto.request
+package org.sopt.and.data.dataremote.model.request
```

Comment:
개인적으로 지금 패키징은 data가 중복되게 두번 사용된거 같아서 dataremote에서 remote로 패기지명을 수정하는것에 대해서는 어떻게 생각하시는지 스윽 질문? 의견?을 남겨봅니다~!
```suggestion
package org.sopt.and.data.remote.model.request
```